components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -18,19 +18,19 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
           <chef-form-field>
             <label>
               <span class="label">ID <span aria-hidden="true">*</span></span>
-              <input chefInput formControlName="id" type="text" (keyup)="handleIDInput($event)" id="id-input">
+              <input chefInput formControlName="id" type="text" (keyup)="handleIDInput($event)" id="id-input" autocomplete="off"/>
             </label>
-            <chef-error *ngIf="createForm?.controls['id'].hasError('maxlength')">
+            <chef-error *ngIf="createForm.get('id').hasError('maxlength') && createForm.get('id').dirty">
               ID must be 64 characters or less.
             </chef-error>
-            <chef-error *ngIf="createForm?.controls['id'].hasError('required')">
+            <chef-error *ngIf="createForm.get('id').hasError('required') && createForm.get('id').dirty">
```

Comment:
did you change these to get rid of the `?` ?

Reply:
The added dirty is the main part, the rest is just making things a bit more consistent.

Reply:
Good eye for detail @bcmdarroch !

In this case, it is OK to delete the safe navigation operator (https://angular.io/guide/template-syntax#the-safe-navigation-operator----and-null-property-paths) because `createForm`, if you trace it back to the consumers, is instantiated in the class constructor, so it is guaranteed to exist when the template is being processed.